src/mono/netcore/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Task/Microsoft.NET.Runtime.MonoAOTCompiler.Task.pkgproj

```diff
@@ -0,0 +1,21 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <PackageDescription>The package that contains the MonoAOTCompiler MSBuild Task</PackageDescription>
```

Comment:
`Provides the MonoAOTCompiler msbuild task`

Reply:
Thanks :)